modules/virt-defining-apps-for-dr.adoc

```diff
@@ -0,0 +1,59 @@
+
+An RHACM-managed application that includes a VM must be created by using a GitOps workflow and by creating an RHACM Application or `ApplicationSet`.
+
+You can configure any other VM in the cluster as an RHACM-discovered application. This includes VMs imported by using the Migration Toolkit for Virtualization (MTV), VMs created by using the {VirtProductName} web console, or VMs created by any other means, such as the CLI. Applications in an RHACM-discovered VM are discovered by the VM through user-defined labels.
+
+[id="best-practices-rhacm-managed-vm_{context}"]
+== Best practices when defining an RHACM-managed VM
+
+There are several actions you can take to improve your experience and chance of success when defining an RHACM-managed VM.
+
+[discrete]
+[id="use-a-pvc-and-populator_{context}"]
+=== Use a PVC and populator to define storage for the VM
+Because data volumes create persistent volume claims (PVCs) implicitly, data volumes and VMs with data volume templates do not fit as neatly into the GitOps model.
+
+[discrete]
+[id="use-import-method_{context}"]
+=== Use the import method when choosing a population source for your VM disk
+The Red Hat registry contains RHEL images. Search the software catalog to see which versions are available. It is recommended to use a specific version of the image rather than a floating tag for consistent results. The KubeVirt community maintains container disks for other operating systems in a Quay repository.
+
+[discrete]
+[id="use-pull-node_{context}"]
+=== Use `pullMethod: node`
+Use the pod `pullMethod: node` when creating a data volume from a registry source to take advantage of the {product-title} pull secret, which is required to pull container images from the Red Hat registry.
+
+[id="best-practices-rhacm-discovered-vm_{context}"]
+== Best practices when defining an RHACM-discovered virtual machine
+
+There are several actions you can take to improve your experience and chance of success when defining an RHACM-discovered VM.
+
+[discrete]
+[id="protect-the-vm_{context}"]
+=== Protect the VM when using MTV, the Openshift Virtualization web console, or a custom VM
+Because automatic labeling is not currently available, the application owner must manually label the components of the VM application when using MTV, the {VirtProductName} web console, or a custom VM.
+
+For VMs created by using data volume templates, add a unique label to to the label section of VM when creating the VM to ensure the same label is applied to the VM's data volumes and PVC.
+
+For VMs referencing an existing, standalone data volume, add a unique label to the label section of the VM when creating the data volume to ensure the same label is applied to the VM's PVC.
+
+For VMs referencing existing, standalone PVCs, add a unique label to the label section when creating the PVC. After the application is defined, you can locate and select the label in the RHACM console and apply a disaster recovery policy.
```

Comment:
This doesn't make sense.  It says "for existing PVCs" add a label when creating the PVC.  Let's just simplify all of this to state:

After the Virtual Machine has been created apply a common label to the following resources associated with the VM: VirtualMachine, DataVolume, PersistentVolumeClaim, Service, Route, Secret, ConfigMap. Do not label VirtualMachineInstances or Pods since these are created and managed automatically by OpenShift Virtualization.

This is the text I plan to use in the KCS.